### ISSUE TITLE
bigip_ltm_policy fails appling new rules when asm enable is changed

### ISSUE THREAD

**Issue by slawekww** [F5Networks/terraform-provider-bigip#737]
### Environment

 * TMOS/Bigip Version: BIG-IP 16.1.3.2 Build 0.0.4 Point Release 2
 * Terraform Version: v1.3.5
 * Terraform bigip provider Version: v1.16.0

### Summary

When Terraform plan has update list of rules in policy, new rules are inserted in order defined in Terraform plan.
However Terraform plan shifts rules and recognizes change `enable -> disable` (or vice versa) in the rules even if there is no change in that rule. It caused Terraform apply with fail error ` Error: HTTP 400 :: {"code":400,"message":"\"enable\" may not be specified with \"disable\"","errorStack":[],"apiError":26214401}`

### Steps To Reproduce

Create TF plan with dynamic policy with dymanic rule list:

```
resource "bigip_ltm_policy" "policy" {
  name     = "/f5-part/f5-policy"
  strategy = "first-match"
  requires = ["http"]
  controls = ["forwarding", "asm"]

  ### IMPORTANT! Please do not change rules order as it matters!
  # Cluster rules
  dynamic "rule" {
    for_each = { for r in var.rules : r.name => r }

    content {
      name = "${rule.value.name}-https"

      action {
        connection = false
        forward    = true
        pool       ="/f5-part/${rule.value.pool}"
      }

      condition {
        http_host = true
        host      = true
        ends_with = true
        request   = true
        values    = ["${rule.value.matching}"]
      }

      action {
        asm        = true
        connection = false
        enable     = ${rule.value.enable}
        policy     = "/f5-part/f5-waf-profile"
      }
    }
  }
}
```

```
rules = [
{
 name = "rule1"
 pool = "pool1"
 matching = "example.com"
 enable = true
},
{
 name = "rule2"
 pool = "pool2"
 matching = "example2.com"
 enable = true
}
]
```

Add more rules to have something like:
```
rules = [
{
 name = "rule1"
 pool = "pool1"
 matching = "example.com"
 enable = true
},
{
 name = "rule-new"
 pool = "pool3"
 matching = "example3.com"
 enable = false
},
{
 name = "rule2"
 pool = "pool2"
 matching = "example2.com"
 enable = true
}
]
```

1. Provide terraform resource config which you are facing trouble along with the output of it.

See above.

3. To get to know more about the issue, provide terraform debug logs

4. To capture debug logs, export TF_LOG variable with debug ( export TF_LOG= DEBUG ) before 
  runnning terraform apply/plan

5. As3/DO json along with the resource config( for AS3/DO resource issues )
n/a, able to define draft policy and publish it manually via F5 with expected change.

### Expected Behavior
Terraform apply should correctly create draft policy and publish it with new rules.

### Actual Behavior
Error is seen during TF apply:  ` Error: HTTP 400 :: {"code":400,"message":"\"enable\" may not be specified with \"disable\"","errorStack":[],"apiError":26214401}`

**Comment by KrithikaChidambaram** [comment]
Hi, thanks for reporting. Internal tracking ID for this issue is: INFRAANO-1048, thanks!

**Comment by RavinderReddyF5** [comment]
Hi @slawekww fixed in v1.17.0
